test/metric_learn_test.py

```diff
@@ -70,6 +73,40 @@ def test_iris(self):
       csep = class_separation(lmnn.transform(), self.iris_labels)
       self.assertLess(csep, 0.25)
 
+  def test_convergence_simple_example(self):
+    # LMNN should converge on this simple example, which it did not with
+    # this issue: https://github.com/metric-learn/metric-learn/issues/88
+    X, y = make_classification(random_state=0)
+    old_stdout = sys.stdout
+    sys.stdout = StringIO()
+    lmnn = LMNN(verbose=True)
```

Comment:
We should use `python_LMNN` here and the other test, to not fail when the shogun version is available.

Reply:
done